doc/getting_started/index.rst

```diff
@@ -318,6 +318,13 @@ to build Zephyr applications.
 
             You cannot move the SDK directory after you have installed it.
 
+      #. Set the required :ref:`environment variable <env_vars>`:
+
+         .. code-block:: bash
+
+            echo 'export ZEPHYR_TOOLCHAIN_VARIANT=zephyr' >> ~/.bashrc
```

Comment:
i use no bash.... Why not . $HOME/.zephyrrc?

Reply:
Yes, that is something I thought about. the problem is that we already use that construct for the PIP path:
https://docs.zephyrproject.org/latest/getting_started/index.html#get-zephyr-and-install-python-dependencies

```
pip3 install --user -U west
echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
source ~/.bashrc
```

so for consistency I used the same mechanism.

Reply:
> $HOME/.zephyrrc?

Because `source zephyr-env.sh` is not even recommended anymore to west users, since it is not required at all.

Reply:
The SDK, when installed, already does the above and sets the variant in the .zephyrrc, so you are just asking people to put it somewhere else now.

As per recommended, I use west, and I heavily depend on .zephyrrc. I can't imagine how I can do my daily work without it and I do not think it was wise to recommend against it. Not everyone uses the zephyr SDK and my .zephyrrc is full of things I need for various toolchains and combos.

Reply:
https://github.com/zephyrproject-rtos/sdk-ng/blob/7ef15bd022c46bd92f12576a5c51dca913507949/scripts/template_dir#L134

Reply:
One option would be to remove the reference to `~/.bashrc` altogether from the main GSG page, and instead just link to the env vars page:
https://docs.zephyrproject.org/latest/guides/env_vars.html#environment-variables

Reply:
>https://github.com/zephyrproject-rtos/sdk-ng/blob/7ef15bd022c46bd92f12576a5c51dca913507949/scripts/template_dir#L134

This is actually no longer needed (the part with `ZEPHYR_SDK_INSTALL_DIR`) because CMake can find that automagically.

>The SDK, when installed, already does the above and sets the variant in the .zephyrrc, so you are just asking people to put it somewhere else now.

OK, I didn't know that. In that case I agree it's better to instruct users to place this in `.zephyrrc`.

> I do not think it was wise to recommend against it

No, we don't recommend against it, it's just we don't emphasize its use anymore

Reply:
> This is actually no longer needed (the part with `ZEPHYR_SDK_INSTALL_DIR`) because CMake can find that automagically.

only if you install in some predefined location, not every can do that. But again, zephyr SDK is not the only toolchain we use, so if our build system got smart with the zephyr sdk, it does not mean we have to drop everything and leaving other toolchains in disadvantage.